nearcore/src/lib.rs

```diff
@@ -148,6 +155,7 @@ pub struct NearNode {
     pub view_client: Addr<ViewClientActor>,
     pub arbiters: Vec<ArbiterHandle>,
     pub rpc_servers: Vec<(&'static str, actix_web::dev::ServerHandle)>,
+    pub cold_store_loop_handle: Option<ColdStoreLoopHandle>,
```

Comment:
add a small comment that this is only set when cold storage is active, and that it copies contents from hot to cold